src/client_side_request.h

```diff
@@ -76,8 +76,13 @@ class ClientHttpRequest
     /// the request. To set the virgin request, use initRequest().
     void resetRequest(HttpRequest *);
 
+    /// update the code in the transaction processing tags
+    void updateLoggingTags(const LogTags_ot code) { al->cache.code.update(code); }
+    /// update error flags in the transaction processing tags
+    void updateLoggingTags(const LogTagsErrors &err) { al->cache.code.err.update(err); }
```

Comment:
Please try to remove this second/overloaded method. AFAICT:

* It is used only twice so we cannot claim that it encapsulates frequently repeated logic.
* In one of those two uses, it triggers creation of more code than a direct ALE update needs.
* Overloading always complicates comprehension/maintenance, at least a bit.
* I am worried about the slippery slope here: Will folks duplicate more/all LogTag methods here if we provide these two? It may get out of hand...
* It is not required because folks can access public ALE directly. The reason we are adding the first updateLoggingTags() method is to encapsulate frequently reused logic and not because folks cannot access ALE.

Reply:
Done (6e7c451).